apps/system/js/app_usage_metrics.js

```diff
@@ -759,6 +767,25 @@
     return dayKey.replace(/-/g, '');
   };
 
+  UsageData.prototype.getSearchCounts = function(provider) {
+    var search = this.data.searches[provider];
+    var dayKey = this.getDayKey();
+    if(!search) {
```

Comment:
nit: add a space between `if` and `(` (a few other places you did this below as well)